src/Legend/legend.css

```diff
@@ -0,0 +1,41 @@
+/*legend styling goes here*/
+
+#legend .title {
+	font-size: 1.3em;
+}
+#legend .subtitle {
+	font-style: italic;
+}
+
+#legend .layer {
+	margin: 10px;
+}
+
+#legend .item .swatch {
+	width: 100px;
+	height: 20px;
+	border: solid 1px;
+	border-color: #d6d5d6
+}
+
+#legend .item.disabled {
+	opacity: 0.5;
+	order: 1;
```

Comment:
Delete this line to keep legend order fixed

Reply:
okay!